conversions/decimal_to_octal.py

```diff
@@ -16,7 +16,7 @@ def decimal_to_octal(num):
         counter += 1
```

Comment:
Let’s add some doctests because this function is not quite right in a few ways...  Please change the top of the function to:
```python
def decimal_to_octal(num: int) -> str:

    """Convert a Decimal Number to an Octal Number.
    
    >>> for i in (0, 2, 8, 64, 65, 216, 255, 256, 512):
    ...     decimal_to_octal(i) == oct(i)
    True
    """
```
Then change line 19 to something like `return f"0o{int(octal)}"` and then do `python3 -m doctest -v conversions/decimal_to_octal.py` to make sure those tests pass.

Reply:
The test case fails, but when I do it manually there is no error as such

Reply:
I changed the code as suggested nevertheless